content/python/concepts/strings/terms/endswith/endswith.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '.endswith()'
+Description: 'Checks whether or not a string ends with a given value.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `.endswith()` method checks a value against a given string and returns `True` if the string ends with that value. Otherwise, it returns `False`.
+
+## Syntax
+
+```pseudo
+string.endswith(value, start, end)
+```
+
+Given a `string`, the `.endswith()` method can be used in the following way:
+
+The `value` is a required argument that is checked if it exists at the end of the `string`.  It is also case-sensitive.
+
+Optionally, the `value` can be tested for whether it is at the end of a particular substring within the `string`. This is done within the `start` and `end` index arguments.
+
+## Example
+
+```py
+example_str = "This is a string"
+
+check_A = example_str.endswith("g")
+check_B = example_str.endswith("s")
+check_C = example_str.endswith("st", 5, 12)
+
+print("A: ", check_A)
+print("B: ", check_B)
+print("C: ", check_C)
+
+# Output A: True
+# Output B: False
+# Output C: True
+```
+
+From the above example, for ```check_A``` the output is True as the ```.endswith()``` function checks whether the the string ```example_str``` ends with the character "g".
```

Comment:
```suggestion
From the above example, for `check_A` the output is True as the `.endswith()` function checks whether the the string `example_str` ends with the character "g".
```

When referencing a piece of code inline, let's surround with just one backtick on each side. Three backticks should be used only for code snippets that span multiple lines.

Reply:
> When referencing a piece of code inline, let's surround with just one backtick on each side. Three backticks should be used only for code snippets that span multiple lines.

Understood, thank you!